### ISSUE TITLE
About test dataset and validate dataset

### ISSUE THREAD

**Issue by gehong-coder** [xiaolan98/BDANN-IJCNN2020#6]
Hello author, in the experimental code of the paper, is the test set you placed also validate? Do you need to change it to test? Did you separate the image_validation data separately? thank you for answering me

**Comment by xiaolan98** [comment]
There are just the training and testing data. The “validate” here is the test set for the Twitter set. I follow the setting of previous work (EANN and MVAE).

**Comment by gehong-coder** [comment]
Thank you so much for answering me so promptly! , I want to ask again, this setting in Twitter
" valiate_data, _ = write_data("validate", image_list, text_only)
test_data, _ = write_data("validate", image_list, text_only)"
Can I follow this setting if I do an experiment?
Is it still used to modify it into this form?
" valiate_data, _ = write_data("validate", image_list, text_only)
test_data, _ = write_data("test", image_list, text_only)"
Thank you very much!

**Comment by xiaolan98** [comment]
Yes, you can follow the original setting. 
But If you think it's better to have the training, validation, and test set, you can split some data from the training set to form a validation set and use the original validation set as the test set.

**Comment by gehong-coder** [comment]
Thank you so much author, you have helped me a lot, much appreciated!
